### DISCUSSION TITLE
How can I create an odf with a mixed texture?

### DISCUSSION THREAD

**Discussion by 1996phd** [mtex-toolbox/mtex#1409] [Ask Anything] [answered]
Hi, everyone!
I am trying to calculate the Taylor factor of the Mg alloy when a mixed texture exists.
The mixed texture contains:
1. with 50%(or other value) a-axis parallel to the x direction;
`V=vector3d(1,0,0)`
`odf = fibreODF(cs.aAxis, V);`
2. and with the remainning grains have their c-axis paraller to the x direction.
`V=vector3d(1,0,0)`
`odf = fibreODF(cs.cAxis, V);`
And my question is how to create such an odf.

**Comment by kilir** [comment, marked as answer]
Hi,
```
0.5* odf1 + 0.5* odf2
```
should do. if proportions differ, just make sure they add to 1.

Cheers,
Rüdiger

**Reply by 1996phd** [reply]
thanks!
